### PR TITLE
optimize: add license header for spring config file

### PR DESCRIPTION
**Pull request by slievrly**
- [ ] I have registered the PR [changes](../changes).

### Ⅰ. Describe what this PR did

add license header for spring config file

### CODE REVIEW

File: .licenserc.yaml
Comment (funky-eyes) [inline .licenserc.yaml:1]: 这个改动会使拥有copyright的其它文件由于格式不同,而被认为没有license header
This change will cause other copyrighted files to be considered without the license header due to their different formats

Comment (funky-eyes) [reply .licenserc.yaml:1]: 应该是我误解了,可能需要#6650及#6651被合并后,才能通过该check
I must have misunderstood, it may take #6650 and #6651 to be merged to pass the check.

Comment (slievrly) [reply .licenserc.yaml:1]: Yes, it need to re-run the CI status after the first two PRs are merged.

**Review by funky-eyes — COMMENTED** [review]
ERROR the following files don't have a valid license header: 
tcc/src/main/resources/META-INF/services/org.apache.seata.integration.tx.api.json.JsonParser 
ERROR one or more files does not have a valid license header

**Review by funky-eyes — APPROVED** [review]
LGTM

**Pull request merged**